specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/virtualWan.json

```diff
@@ -3413,183 +3413,651 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the resource group."
+            "description": "The resource group name of the VirtualHub."
           },
           {
-            "name": "expressRouteGatewayName",
+            "name": "virtualHubName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the ExpressRoute gateway."
+            "description": "The name of the VirtualHub."
           },
           {
-            "name": "connectionName",
+            "name": "ipConfigName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the connection subresource."
-          },
-          {
-            "name": "putExpressRouteConnectionParameters",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/ExpressRouteConnection"
-            },
-            "description": "Parameters required in an ExpressRouteConnection PUT operation."
+            "description": "The name of the ipconfig."
           },
           {
             "$ref": "./network.json#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
           }
         ],
         "responses": {
           "200": {
-            "description": "Update successful. The operation returns the ExpressRouteConnection.",
-            "schema": {
-              "$ref": "#/definitions/ExpressRouteConnection"
-            }
-          },
-          "201": {
-            "description": "Create successful. The operation returns the ExpressRouteConnection.",
+            "description": "Request successful. Returns the details of the Bgp Connection retrieved.",
             "schema": {
-              "$ref": "#/definitions/ExpressRouteConnection"
+              "$ref": "#/definitions/HubIpConfiguration"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "Error.",
             "schema": {
-              "$ref": "./network.json#/definitions/CloudError"
+              "$ref": "./network.json#/definitions/Error"
             }
           }
-        },
-        "x-ms-long-running-operation": true,
-        "x-ms-long-running-operation-options": {
-          "final-state-via": "azure-async-operation"
         }
       },
-      "get": {
-        "tags": [
-          "ExpressRouteConnections"
-        ],
-        "operationId": "ExpressRouteConnections_Get",
+      "put": {
+        "operationId": "VirtualHubIpConfiguration_CreateOrUpdate",
         "x-ms-examples": {
-          "ExpressRouteConnectionGet": {
-            "$ref": "./examples/ExpressRouteConnectionGet.json"
+          "VirtualHubRouteTableV2Put": {
```

Comment:
Should it be VirtualHubIpConfigurationPut?

Reply:
Done. Thanks!